src/main/io/vtx_smartaudio.c

```diff
@@ -1036,7 +1036,7 @@ static uint8_t vtxSAGetPowerLevels(const vtxDevice_t *vtxDevice, uint16_t *level
 
     for (uint8_t i = 0; i < saSupportedNumPowerLevels; i++) {
         levels[i] = saSupportedPowerValues[i];
-        uint16_t power = (uint16_t)pow(10.0,levels[i]/10.0);
+        uint16_t power = (uint16_t)powf(10.0f, levels[i] / 10.0f);
```

Comment:
We don't need `10.0f` because of `-fsingle-precision-constant`

Reply:
Yeah this is just for consistency as I think we generally use the float suffix? Not sure what is the preferred style.

Reply:
Yes afaik the BF standard is to explicitly go with float syntax.

Reply:
Then we can make PR for 4.4 with it.